packages/@vuepress/bundler-webpack/src/dev/createDevServerConfig.ts

```diff
@@ -28,11 +28,11 @@ export const createDevServerConfig = (
     },
     host: app.options.host,
     hot: true,
-    onAfterSetupMiddleware: (expressApp, server) => {
+    onAfterSetupMiddleware: ({ app: expressApp }, server) => {
       // plugin hook: afterDevServer
       options.afterDevServer?.(expressApp, server)
     },
-    onBeforeSetupMiddleware: (expressApp, server) => {
+    onBeforeSetupMiddleware: ({ app: expressApp }, server) => {
       // use trailing slash middleware to support vuepress routing in dev-server
       // it will be handled by most of the deployment platforms
       expressApp.use(trailingSlashMiddleware)
```

Comment:
Why this line does not throw error?

Reply:
This is the expect express app instance which webpack dev server use.

Working in my local env.